main.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 //VERSION of ergo.
-var VERSION string
+var VERSION = "unofficial build"
```

Comment:
Could we add some comment explaining why it is has this version? 
```
// VERSION of ergo.
// When ergo is build without a proper tag/release it is named as `unofficial version`
// For instance, installing through `go get github.com/cristianoliveira/ergo`
```

Reply:
Of course, in my opinion, it is reasonable